### PR TITLE
[FEATURE] New randoms system

### PR DESCRIPTION
**Pull request by SandroCoutinho**
Related to issue #17.

### CODE REVIEW

File: src/main/java/org/parabot/randoms/pkhonor/MysteriousOldMan.java [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
 /**
- * Created with IntelliJ IDEA. User: Piet Jetse Date: 12-9-2014 Time: 17:19
+ * @author Fryslan
  */
 public class MysteriousOldMan implements Random {
 
     private Npc man;
-    private final int id = 410;
+    private final int ID = 410;
```
Comment (JKetelaar) [inline src/main/java/org/parabot/randoms/pkhonor/MysteriousOldMan.java]: Should be a lowercase constant, it's not public

File: src/main/java/org/parabot/randoms/pkhonor/SandwichLady.java [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 
 /**
- * Created with IntelliJ IDEA. User: Piet Jetse Date: 12-9-2014 Time: 16:13
+ * @author Fryslan
  */
 public class SandwichLady implements Random {
 
     private Npc lady;
-    private final int id = 3117;
+    private final int ID = 3117;
```
Comment (JKetelaar) [inline src/main/java/org/parabot/randoms/pkhonor/SandwichLady.java]: Lowercase constant again

File: src/main/java/org/parabot/randoms/pkhonor/TriangleSandwich.java [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
 
 /**
- * Created with IntelliJ IDEA. User: Piet Jetse Date: 11-9-2014 Time: 22:29
+ * @author Fryslan
  */
 public class TriangleSandwich implements Random {
 
-    Item item;
+    private Item item;
+    private final int ID = 6963;
```
Comment (JKetelaar) [inline src/main/java/org/parabot/randoms/pkhonor/TriangleSandwich.java]: Lowercase constant again

**Comment by SandroCoutinho** [comment]
Woops my bad.

**Pull request merged**